hivemind/moe/server/dht_handler.py

```diff
@@ -16,21 +16,24 @@
     split_uid,
 )
 from hivemind.p2p import PeerID
-from hivemind.utils import MPFuture, get_dht_time
+from hivemind.utils import MAX_DHT_TIME_DISCREPANCY_SECONDS, MPFuture, get_dht_time
 
 
 class DHTHandlerThread(threading.Thread):
-    def __init__(self, experts, dht: DHT, update_period: int = 5, **kwargs):
+    def __init__(self, experts, dht: DHT, update_period: int = 30, expiration: Optional[int] = None, **kwargs):
```

Comment:
If I'm not mistaken, it is not possible to pass `expiration` into this class for a default usage, since the corresponding constructor call in server.py [does not pass](https://github.com/learning-at-home/hivemind/blob/default_expiration/hivemind/moe/server/server.py#L74-L78) the expiration parameter. Is this intentional? If we have reports about making this hard to configure, maybe we should provide a way to configure this expiration explicitly